src/silx/gui/widgets/OverlayMixIn.py

```diff
@@ -45,6 +64,46 @@ def setParent(self, parent: qt.QWidget):
         super().setParent(parent)
         self._registerParent(parent)
 
+    def _getGeometryTopLeft(self) -> tuple[int]:
+        """Return the top left corner of the geometry to set up the geometry"""
+
+        parent = self.parent()
+
+        overlay_size: qt.QSize = self.sizeHint()
+        if isinstance(parent, PlotWidget):
+            offset = parent.getWidgetHandle().mapTo(parent, qt.QPoint(0, 0))
+            canvas_left, canvas_top, canvas_width, canvas_height = (
+                parent.getPlotBoundsInPixels()
+            )
+            canvas_left += offset.x()
+            canvas_top += offset.y()
+        else:
+            canvas_width = parent.size().width()
+            canvas_top = canvas_height = parent.size().height()
+            canvas_left = 0
+            canvas_top = 0
+
+        # calculate left position
+        if self._alignment & qt.Qt.AlignTop:
```

Comment:
Does this works with PyQt5, PyQt6 and PySide6? That's the kind of stuff that changed AFAIR

Reply:
Ah sh*** I will test the different version

Reply:
It seems to work the same for PyQt5 and PySide6 at least...

Reply:
Great, then!
Thanks for checking!